task1/css_form.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Регистрация </title>
+    <link href="css_form.css" rel="stylesheet">
+</head>
+
+<body>
+    <form name="form" method="post" action="register.php" class="container">
+        <legend> регистрация </legend>
+        <div id="container">
+            <input type="email" id="Email" name="Email" minLength="3" maxLength="10" placeholder="E-mail" required
+                pattern="\w+" title="Email with length between 3 and 10. Allowed chars: letters, digits and _">
+
+            <input type="text" id="name" name="name" minLength="3" maxLength="10" placeholder="Име" required
+                pattern="\w+" title="Name with length between 3 and 10. Allowed chars: letters">
+
+            <input type="text" id="lastName" name="lastName" minLength="3" maxLength="10" placeholder="Фамилия" required
+                pattern="\w+" title="Ласт нame with length between 3 and 10. Allowed chars: letters">
```

Comment:
You may want to stick to English letters here maybe?
```suggestion
                pattern="\w+" title="Last name with length between 3 and 10. Allowed chars: letters">
```